### PR TITLE
[DON'T merge] vdpa/virtio: move notify control before device conf

### PR DESCRIPTION
**Pull request by yajwu**

### CODE REVIEW

File: lib/eal/common/eal_common_log.c [hunk 1]
```diff
@@ -500,6 +502,8 @@ rte_vlog(uint32_t level, uint32_t logtype, const char *format, va_list ap)
 	RTE_PER_LCORE(log_cur_msg).loglevel = level;
 	RTE_PER_LCORE(log_cur_msg).logtype = logtype;
 
+	gettimeofday(&_now, NULL);
```
Comment (kailiangz1) [inline lib/eal/common/eal_common_log.c:505]: will this influence the performance ?

File: drivers/vdpa/virtio/virtio_vdpa.c [hunk 2]
```diff
@@ -533,19 +534,6 @@ virtio_vdpa_dev_notifier(void *arg)
 	}
 	DRV_LOG(INFO, "%s vid %d dev notifier work of vq id:%d finish",
 			work->priv->vdev->device->name, work->priv->vid, work->vq_idx);
-	/* Notify device anyway, in case loss doorbell */
```
Comment (kailiangz1) [inline drivers/vdpa/virtio/virtio_vdpa.c:536]: this might needed if notify comes late, we should kick again

Comment (yajwu) [reply drivers/vdpa/virtio/virtio_vdpa.c:536]: This is just for debug, will remove.

File: lib/eal/common/eal_common_log.c (hunk 1 above)
Comment (yajwu) [reply lib/eal/common/eal_common_log.c:505]: This is just for debug, will remove.

File: drivers/vdpa/virtio/virtio_vdpa.c (hunk 2 above)
Comment (yajwu) [reply drivers/vdpa/virtio/virtio_vdpa.c:536]: Yes, will add read kickfd and if need kick device after priv->configured.

**Pull request closed without merging**